llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2376,6 +2327,59 @@ class VPReductionRecipe : public VPRecipeWithIRFlags {
   }
 };
 
+/// A recipe for forming partial reductions. In the loop, an accumulator and
+/// vector operand are added together and passed to the next iteration as the
+/// next accumulator. After the loop body, the accumulator is reduced to a
+/// scalar value.
+class VPPartialReductionRecipe : public VPReductionRecipe {
```

Comment:
Should the `classof` for `VPReductionRecipe` now include `VPPartialReductionRecipe`?

Reply:
Done.